google/cloud/dataproc_spark_connect/session.py

```diff
@@ -560,6 +568,113 @@ def stop(self) -> None:
             ):
                 DataprocSparkSession._active_session.session = None
 
+    # The code to get styles and a custom button is taken from -
+    # https://github.com/googleapis/python-aiplatform/blob/main/google/cloud/aiplatform/utils/_ipython_utils.py#L47
+    @staticmethod
```

Comment:
Can we use Google's utility but catch exception if it's not available in non-Google IDEs?